### PR TITLE
Remove redundant if

### PR DESCRIPTION
**Pull request by kopiczko**
Towards https://github.com/giantswarm/giantswarm/issues/12061

### CODE REVIEW

File: cmd/sync/runner.go [hunk 1]
```diff
@@ -152,35 +152,33 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 				continue
 			}
 
-			if !tagExists {
```
Comment (corest) [inline cmd/sync/runner.go:155]: I didn't get why this is removed

Comment (kopiczko) [reply cmd/sync/runner.go:155]: https://github.com/giantswarm/crsync/pull/20/files#diff-7dda0647dafafce63449dfa490b21807R152

Comment (corest) [reply cmd/sync/runner.go:155]: :facepalm:

**Comment by kopiczko** [comment]
Covered in https://github.com/giantswarm/crsync/pull/21

**Pull request closed without merging**